sdk/samples/iot/README.md

```diff
@@ -69,8 +69,10 @@ To view scenario-focused examples using the API calls, please view the Azure IoT
 You can use Github Codespaces to be up and running quickly! Here are the steps to follow (assuming you already have an IoT Hub set up).
 
 1. Select the "Open with Codespaces" prompt on Github and then "New codespace".
-    ![codespace](./docs/img/codespace.png)
-1. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
+
+<img src="./docs/img/codespace.png" width="80%">
+
+2. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
```

Comment:
nit: this should be a 1. and MD automatically takes care of it

Reply:
```suggestion
1. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
```